### PR TITLE
Keep Metrics and Logs during Network deletion

### PR DESCRIPTION
**Pull request by narayanprusty**

### CODE REVIEW

File: network/delete.go [hunk 1]
```diff
@@ -14,28 +18,66 @@ func Delete() error {
...
+			if strings.Contains(deployment.Name, "miner") || strings.Contains(deployment.Name, "poet") {
+				err := kubernetes.Client.AppsV1().Deployments("default").Delete(context.TODO(), deployment.Name, metav1.DeleteOptions{})
+
+				if err != nil {
+					return err
+				}
+			}
+		}
+
+		err = kubernetes.Client.CoreV1().Namespaces().Delete(context.TODO(), "ws", metav1.DeleteOptions{})
```
Comment (moshababo) [inline network/delete.go:70]: Are you sure that this finalizes everything we need to delete (which doesn't logs/metrics)?

File: config/config.go [hunk 2]
```diff
@@ -62,6 +62,7 @@ type Configuration struct {
 	SmappVersion             string `mapstructure:"smapp-version"`
 	TLSKey                   string `mapstructure:"tls-key"`
 	TLSCert                  string `mapstructure:"tls-cert"`
+	KeepLogsMetrics          bool   `mapstructure:"keep-logs-metrics"`
```
Comment (moshababo) [inline config/config.go:65]: How do we suppose to delete logs/metrics in the aftermath? If manually, then a separate command solely for that should probably be added.

File: network/delete.go (hunk 1 above)
Comment (narayanprusty) [reply network/delete.go:70]: Yes I have tested this.

File: config/config.go (hunk 2 above)
Comment (narayanprusty) [reply config/config.go:65]: You can then use the deleteNetwork sub command without --keep-logs-metrics CLI option. It will delete everything.

**Review by moshababo — APPROVED** [review]

**Pull request merged**